Src/AutoFixtureUnitTest/FixtureTest.cs

```diff
@@ -4904,6 +4904,25 @@ public void CustomizingEnumerableCustomizedCreateManyWhenCreateManyIsMappedToEnu
             Assert.Equal(expected, actual.ToArray());
         }
 
+        [Fact]
+        public void ItIsPossibleToMapManyRequestToCustomEnumerableInstance()
+        {
+            // Arrange
+            var sut = new Fixture();
+            var expected = new[] { "a", "b", "c", "d" };
+            sut.Customizations.Add(
+                new FilteringSpecimenBuilder(
+                    new FixedBuilder(expected),
+                    new EqualRequestSpecification(
+                        new MultipleRequest(new SeededRequest(typeof(string), default(string))))));
```

Comment:
`new SeededRequest` should be in its own line since all the other `new` calls are on their own lines too.

Reply:
Fixed!